ttnn/cpp/pybind11/operations/binary.hpp

```diff
@@ -51,34 +53,47 @@ void bind_binary_operation(py::module& module, const binary_operation_t& operati
         module,
         operation,
         doc,
+        // tensor and scalar
         ttnn::pybind_overload_t{
             [](const binary_operation_t& self,
                const ttnn::Tensor& input_tensor_a,
                const float scalar,
                const std::optional<ttnn::MemoryConfig>& memory_config,
                const std::optional<const DataType>& dtype,
-               const std::optional<std::vector<std::string>>& activations) -> ttnn::Tensor {
-                return self(input_tensor_a, scalar, memory_config, dtype, activations);
+               const std::optional<ttnn::Tensor>& output_tensor,
+               const std::optional<std::vector<std::string>>& activations,
+               const uint8_t& queue_id) -> ttnn::Tensor {
+                return self(queue_id, input_tensor_a, scalar, memory_config, dtype, output_tensor, activations);
```

Comment:
I thought we were going to make the queue_id in the front here?  Why is it the last arg?

Reply:
There are 2 versions in C++ for each of eq(tensor_a, tensor_b) vs eq(tensor_a, scalar): 
- with queue_id in front
- w/o

There is a single version in Py for each 
- via kwarg